### PR TITLE
[VEN-729] Contracts Overview

### PR DESCRIPTION
**Pull request by coreyar**
Adds an overview page to give a high level picture of Isolated Lending contracts

### CODE REVIEW

File: getting-started/contracts-overview.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# Contracts Overview
+
+## Contracts Overview
+
+Venus Protocol V4 contracts are divided in three repositories:
+
+* [isolated-pools](https://github.com/VenusProtocol/isolated-pools): Contains core contracts for isolated lending V4, including logic for suppling, borrowing, liquidations, pool and market deployments, and interest rate models.
+* [oracle](https://github.com/VenusProtocol/oracle): This repo has contracts for oracles that we support as well as logic for validating prices returned from those oracles.
+
```
Comment (kkirka) [inline getting-started/contracts-overview.md:10]:
```suggestion
* [venus-protocol](https://github.com/VenusProtocol/venus-protocol): This repo contains contracts for the Core Pool, which are a bit different from the ones used for isolated lending.

```

File: getting-started/contracts-overview.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
+# Contracts Overview
+
+## Contracts Overview
+
+Venus Protocol V4 contracts are divided in three repositories:
```
Comment (kkirka) [inline getting-started/contracts-overview.md:5]: Or two?

Comment (coreyar) [reply getting-started/contracts-overview.md:5]: I was going to add a summary of the `venus-protocol` contracts as well

Comment (coreyar) [reply getting-started/contracts-overview.md:5]: Updated

File: getting-started/contracts-overview.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+
+**Lending Contracts**
+
+#### [Comptroller](../core-pool/comptroller.md)
+At the heart of the Core Pool is the comptroller. The latever version is [ComptrollerG5](https://github.com/VenusProtocol/venus-protocol/blob/develop/contracts/ComptrollerG5.sol). Previous versions are also kept in the repo. The comptroller is responsibile for listing markets, managing user's positions in markets, liquidations, and emitting rewards. It contains setters and getters for market configuration variables such as collateral factor, close factor, and liquidation incentive. Lending actions can be be paused globally or per market from the comptroller.
+
+#### [JumpRateModel](../core-pool/jump-model.md)
+Each market gets deployed with an interest rate model. The [JumpRateModel](https://github.com/VenusProtocol/venus-protocol/blob/develop/contracts/JumpRateModel.sol) uses a linear curve to determain interest rates based on supply and demand of the asset until it reaches the kink after which there is a sharp increase in rates.
```
Comment (kkirka) [inline getting-started/contracts-overview.md]:
```suggestion
Each market gets deployed with an interest rate model. The [JumpRateModel](https://github.com/VenusProtocol/venus-protocol/blob/develop/contracts/JumpRateModel.sol) uses a linear curve to determine interest rates based on supply and demand of the asset until it reaches the kink after which there is a sharp increase in rates.
```

File: getting-started/contracts-overview.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+
+#### [GovernorBravoDelegate](../governance/bravo.md)
+
+The core logic for governance proposals is in the [GovernorBraveDelegate](https://github.com/VenusProtocol/venus-protocol/blob/main/contracts/Governance/GovernorBravoDelegate.sol) contract. It enables submitting proposals, moving proposals through time-gated stages, canceling and executing proposals as well as voting logic. The voting threshold as well as timelocks are set on this contract.
+
+
+**Lending Contracts**
+
+#### [Comptroller](../core-pool/comptroller.md)
+At the heart of the Core Pool is the comptroller. The latever version is [ComptrollerG5](https://github.com/VenusProtocol/venus-protocol/blob/develop/contracts/ComptrollerG5.sol). Previous versions are also kept in the repo. The comptroller is responsibile for listing markets, managing user's positions in markets, liquidations, and emitting rewards. It contains setters and getters for market configuration variables such as collateral factor, close factor, and liquidation incentive. Lending actions can be be paused globally or per market from the comptroller.
```
Comment (kkirka) [inline getting-started/contracts-overview.md]: `Comptroller` is the latest, all the G*s are ancient. We only kept them because they were used in tests

**Review by Debugger022 — APPROVED** [review]

**Pull request merged**